### PR TITLE
MM-1618 Adds help text to the Get Link modal that informs user when link has been copied to clipboard

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Small green text toast pops up next to the "Copy Link" button informing the user that the team invite link has been copied to the clipboard.  Also did minor cosmetic refactoring to better fit style guide

### CODE REVIEW

**Comment by crspeller** [comment]
Looks Good to me.

File: web/react/components/get_link_modal.jsx [hunk 1]
```diff
@@ -10,46 +10,57 @@ ZeroClipboardMixin.ZeroClipboard.config({
...
-              self.setState({title: $(button).attr('data-title'), value: $(button).attr('data-value') });
+              self.setState({title: $(button).attr('data-title'), value: $(button).attr('data-value')});
+          });
+          $(this.refs.modal.getDOMNode()).on('hide.bs.modal', function() {
+              self.setState({copiedLink: false});
           });
         }
     },
     getInitialState: function() {
         return { };
```
Comment (nickago) [inline web/react/components/get_link_modal.jsx]: Probably want to add an entry for copiedLink to the Initial State

Comment (rgarmsen2295) [inline web/react/components/get_link_modal.jsx]: Yup will do.

**Comment by rgarmsen2295** [comment]
Added the copiedLink field to the initial state, please take another look, @nickago

**Comment by nickago** [comment]
Sweet, +1

**Pull request merged**